GooglePlacesAutocomplete.js

```diff
@@ -644,13 +644,19 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
         showsVerticalScrollIndicator={false}
         focusable={false}
       >
-        <TouchableHighlight
-          style={
-            props.isRowScrollable ? { minWidth: '100%' } : { width: '100%' }
-          }
+        <Pressable
+          style={({ hovered, pressed }) => [
+            props.isRowScrollable ? { minWidth: '100%' } : { width: '100%' },
+            {
+              backgroundColor: pressed
+                ? props.listUnderlayColor
+                : hovered
+                ? props.listHoverColor
```

Comment:
Since these become optional, we should give some default highlight and hover colors. 

@grgia right?

Reply:
Default `listHoverColor` was already set [here](https://github.com/Expensify/react-native-google-places-autocomplete/blob/3bbd17d63e6c38d38d857b50f6037c1c0376ff06/GooglePlacesAutocomplete.js#L941). But `listUnderlayColor` not set.
This matches current app version with light background on press, no background on hover.
I can add default listHoverColor if needed. Just need color value.

Reply:
@situchan I think adding alpha 80% for listUnderlayColor`#c8c7cc` should be good. @grgia any thoughts?

Reply:
My suggestion

#c8c7cc - underlay
#ececec - hover

Reply:
@grgia fine with these default color values?

Reply:
Taking a look, will update in a minute!

Reply:
Yep, I think these are fine defaults.

Reply:
@grgia @Santhosh-Sellavel updated